src/main/java/uk/gov/hmcts/reform/unspec/model/CaseData.java

```diff
@@ -88,4 +88,5 @@
     private final CloseClaim discontinueClaim;
 
     private final BusinessProcess businessProcess;
+    private final String stateFlowState;
```

Comment:
I don't think we need to preserve flow state, it will be calculated based on the CaseData.

Reply:
this is removed now